src/settings.rs

```diff
@@ -237,7 +237,7 @@ pub mod log_level {
     use std::fmt;
 
     #[derive(Debug, Clone, Copy)]
-    pub struct Level(tracing::Level);
+    pub struct Level(pub tracing::Level);
```

Comment:
Why do you need this? I’d use a From trait or Deref one to get here. But since you’re not using what you changed?

Reply:
This allows constructing `Settings` without needing to go through the full file parsing.